### ISSUE TITLE
Qt offering changes 2020

### ISSUE THREAD

**Issue by mseliger** [dvorka/mindforger#1186]
In future Long-term-supported (LTS) releases and the offline installer of Qt will become available to commercial licensees only (Version 5.15).
https://www.qt.io/blog/qt-offering-changes-2020
Will this be a problem for the development of Mindforger?

**Comment by dvorka** [comment]
Great question Maria!

TL:DR I believe that it is not and will not be problem for MindForger.

MindForger is in good position from the open source requirements perspective - it's GPL, no $/sponsor/gift and basically one man show. Situation may slightly differ by platform:

* **Linux**: as FLOSS believer I trust Linux community and ecosystem. Qt non-LTS releases and source code will be available. Distributions and individuals will either go with rolling releases and/or build binaries. It's also probable that there will be fork(s) to fix non-LTS releases bugs driven by Qt community. In other words, I hope that thanks to KDE and OSS Qt applications MindForger is safe.
 
* Windows and macOS: Situation might be slightly different. Brew is hope for macOS. Will see about Win. Anyway there is fallback - building Qt from sources. It just matter of time and priorities (for me).

Thank you that you care, Thank you for your interest in MindForger!

---

See  also:
* KDE is going to be fine: https://tsdgeos.blogspot.com/2020/01/the-qt-company-is-stopping-qt-lts.html
* Krita: https://valdyas.org/fading/software/about-qt-offering-changes-2020/
* Qt announcement changes, history and communication: https://www.reddit.com/r/programming/comments/euopv6/qt_offering_changes_2020_mandatory_registration/
